packages/scalecube-microservice/src/ServiceCall/RemoteCall.ts

```diff
@@ -26,7 +28,71 @@ export const remoteCall = ({
     }) as Observable<any>;
   }
 
-  // TODO remote invoke
-  // TODO if service is remote then use transport else invoke the function
-  return of({});
+  return remoteResponse({ address: endPoint.address, asyncModel, message, openConnections });
+};
+
+const remoteResponse = ({
+  address,
+  asyncModel,
+  message,
+  openConnections,
+}: {
+  address: string;
+  asyncModel: string;
+  message: Message;
+  openConnections: { [key: string]: any };
+}) => {
+  let connection: any;
+  if (!openConnections[address]) {
+    const client = CreateClient({ address });
+    connection = client.connect();
+    openConnections[address] = connection;
+  } else {
+    connection = openConnections[address];
+  }
+
+  return new Observable((observer) => {
+    try {
```

Comment:
why are you using try/catch for promise handling? seems like catch for the promise in this case is appropriate